packages/design-system/src/components/Autocomplete/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as Autocomplete } from './Autocomplete';
+export { default as AutocompleteWrapperDiv } from './WrapperDiv';
```

Comment:
Hm, do we want to expose the `AutocompleteWrapperDiv` to applications? It's currently just needed by the core ds Autocomplete right?

Reply:
Will it still generate the type definition if it is not exported here? Only asking because I deleted the type file previously found at `packages/design-system/src/types/Autocomplete/WrapperDiv.d.ts`

Although... maybe I should keep the naming consistent?

Reply:
Just tried removing the `export WrapperDiv` line from `packages/design-system/src/components/Autocomplete/index.ts` and looks like it works as the WrapperDiv typescript definitions is being generated by the cmsds scripts (eg. check for `packages/design-system/dist/types/Autocomplete/WrapperDiv.d.ts)